### PR TITLE
New ORM library named Typetta

### PR DESCRIPTION
**Pull request by minox86**
i propose to add a new library named Typetta.

### CODE REVIEW

**Comment by minox86** [comment]
I completed the corporate CLA, so I'll wait for approval.

**Comment by minox86** [comment]
Is there something that I can do to help approving this PR? Please let me know. Thx

**Comment by orta** [comment]
Thanks for the ping, but this project doesn't seem to be a service related to GraphQL?

**Comment by minox86** [comment]
Please, check it another time. Typetta heavily rely on GraphQL to define the data model and offers a lot of functionalities for GraphQL developers. It defenitely makes sense only in a GraphQL environment. Anyway it's all documented here:

https://twinlogix.github.io/typetta/chapters/advanced/graphql
https://twinlogix.github.io/typetta/chapters/data-model/entities

The list on this repo includes many other ORMs and they are far less connected to GraphQL, in my opinion.

**Comment by minox86** [comment]
I'm really sorry to insist, but is there someone who can give me a feedback considering the elements I provided? 
Thank you very much.

File: src/content/code/services/typetta.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Typetta
+description: Typetta is an open-source ORM written in TypeScript that aims to allow seamless access to data in a typed fashion to all main SQL databases (MySQL, PostgreSQL, Microsoft SQL Server, SQLLite3, CockroachDB, MariaDB, Oracle e Amazon Redshift) and also to the NoSQL database MongoDB. For more information, go to https://twinlogix.github.io/typetta/
```
Comment (orta) [inline src/content/code/services/typetta.md]:
```suggestion
description: Typetta is an open-source ORM written in TypeScript that aims to allow seamless access to data in a typed fashion to all main SQL databases (MySQL, PostgreSQL, Microsoft SQL Server, SQLLite3, CockroachDB, MariaDB, Oracle * Amazon Redshift) and also to the NoSQL database MongoDB.
```

Comment (orta) [reply src/content/code/services/typetta.md]: I've cleaned up the English a bit and removed the redundant link - this should be good to go after that,  thanks for providing more context

Comment (minox86) [reply src/content/code/services/typetta.md]: Thanks you very much for your time and attention Orta!

File: src/content/code/services/typetta.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Typetta
+description: Typetta is an open-source ORM written in TypeScript that aims to allow seamless access to data in a typed fashion to all main SQL databases (MySQL, PostgreSQL, Microsoft SQL Server, SQLLite3, CockroachDB, MariaDB, Oracle * Amazon Redshift) and also to the NoSQL database MongoDB.
```
Comment (orta) [inline src/content/code/services/typetta.md]:
```suggestion
description: Typetta is an open-source ORM written in TypeScript that aims to allow seamless access to data in a typed fashion to all main SQL databases (MySQL, PostgreSQL, Microsoft SQL Server, SQLLite3, CockroachDB, MariaDB, Oracle & Amazon Redshift) and also to the NoSQL database MongoDB.
```

**Pull request merged**